.gitignore

```diff
@@ -11,3 +11,5 @@ parts/
 doc/_build
 htmlcov/
 .coverage
+coverage.xml
+nosetests.xml
```

Comment:
Yeah, it's not needed any more.

(But I thought having it  in .gitignore doesn't hurt either and felt too lazy to do anything about it.)